usr/share/rear/lib/bootloader-functions.sh

```diff
@@ -506,8 +506,6 @@ set default="$GRUB2_DEFAULT_BOOT"
 
 insmod efi_gop
 insmod efi_uga
```

Comment:
even `efi_gop` and `efi_uga` can be eliminated, `all_video` depends on them.

Reply:
@pcahyna 
can I find out on a non UEFI system which GRUB2 modules
depend on each other so that I could verify things on SUSE?
I have only the plain GRUB2 module files from a grub2-x86_64-efi RPM.

Reply:
According to
https://github.com/rear/rear/issues/2388#issuecomment-625752647
the GRUB2 modules `all_video` `efi_gop` and `efi_uga`
are always there for SLE15, SLE12, and SLE11.

So things should work for now when `all_video` and `efi_gop` and `efi_uga`
are enforced loaded.

Because I am not at all a sufficient GRUB2 expert
to make an educated decision if it is better to
enforce loading `all_video` and `efi_gop` and `efi_uga`
or only enforce loading `all_video`
I would from my current point of view a short time before ReaR 2.6 release
perfer to minimize the changes to avoid unexpected regressions
and merge it as it is currently.

A generic cleanup and enhancement of the code that does
"GRUB2 install as bootloader for the recovery system"
cf. https://github.com/rear/rear/issues/2388
is needed anyway but that should happen after the ReaR 2.6 release.

Reply:
@jsmeix 
> can I find out on a non UEFI system which GRUB2 modules
> depend on each other so that I could verify things on SUSE?
> I have only the plain GRUB2 module files from a grub2-x86_64-efi RPM.

For me it works to download the package containing the modules (grub2-efi-x64-modules*.rpm for Fedora-based distros) and peek into `moddep.lst` ( `/usr/lib/grub/x86_64-efi/moddep.lst` in my case). It has:
```
all_video: efi_gop efi_uga video_bochs video_cirrus
```

Reply:
@pcahyna 
thank you for your help!
I will have a look at what `moddep.lst` contains for SUSE.

Reply:
Also the GRUB2 modules efi_gop and efi_uga are no longer loaded, see https://github.com/rear/rear/issues/2388#issuecomment-628035191 and
https://github.com/rear/rear/commit/2470edf534bc836a3f13d8c48e3007a24bb63bf4